packages/core/src/components/editable-text/_editable-text.scss

```diff
@@ -31,6 +31,12 @@
     box-shadow: input-transition-shadow($input-shadow-color-focus, true), $input-box-shadow-focus;
   }
 
+  @media (forced-colors: active) and (prefers-color-scheme: dark) {
+    &::before {
```

Comment:
I think we should add disabled styles as well; we don't want to keep showing a border in this case:

<img width="762" alt="image" src="https://user-images.githubusercontent.com/723999/192382416-331d7875-f37a-4255-b538-6025ceb94e2c.png">